### PR TITLE
D5

### PR DESCRIPTION
**Pull request by val-fed**

### CODE REVIEW

File: file.txt [hunk 1]
```diff
@@ -14,12 +14,14 @@ line 13
...
-line 17
+    System.out.println("\n");
 line 18
 line 19
 line 20
 line 21
-line 22
+
+
+System.out.println("\n");
```
Comment (val-fed) [inline file.txt]: 24 R

File: file.txt [hunk 2]
```diff
@@ -14,12 +14,14 @@ line 13
...
 line 23
 line 24
```
Comment (val-fed) [inline file.txt]: 24 L

File: file.txt (hunk 1 above)
Comment (valerii-fed) [inline file.txt]: Created by: **valerii.fedorenko.ua@gmail.com**
Time: **Sun Jan 18 22:31:47 EET 1970**
24 R

Comment (valerii-fed) [inline file.txt]: Created by: **valerii.fedorenko.ua@ukr.net**
Time: **Sun Jan 18 22:31:53 EET 1970**
Created by: **valerii.fedorenko.ua@gmail.com**
Time: **Sun Jan 18 22:31:47 EET 1970**
24 R

File: file.txt (hunk 2 above)
Comment (valerii-fed) [inline file.txt]: Created by: **valerii.fedorenko.ua@gmail.com**
Time: **Sun Jan 18 22:31:47 EET 1970**
24 L

Comment (valerii-fed) [inline file.txt]: Created by: **valerii.fedorenko.ua@ukr.net**
Time: **Sun Jan 18 22:31:53 EET 1970**
Created by: **valerii.fedorenko.ua@gmail.com**
Time: **Sun Jan 18 22:31:47 EET 1970**
24 L

**Pull request closed without merging**